Source/WebKit/glib/remote-inspector.md.in

```diff
@@ -0,0 +1,64 @@
+Title: Remote Web Inspector
+Slug: remote-inspector
+
+# Remote Web Inspector
+
+The remote Web Inspector enables debugging of web pages from a different
+device. This may be particularly useful in environments where it may not be
+feasible to use the Web Inspector in the same screen, or when input devices
+(keyboard, mouse) are not available.
+
+#if WPE
+
+Note WPE WebKit only supports using the Web Inspector remotely, as embedded
+devices often lack input devices, have smaller screens, use custom windowing
+systems, or may even be headless (no screen at all).
```

Comment:
This is not accurate, WPE with the new api supports the local inspector.

Reply:
Good point, let's replace `s/only/always/`, and add a little note:

```
Note WPE WebKit always supports using the Web Inspector remotely, as embedded
devices often lack input devices, have smaller screens, use custom windowing
systems, or may even be headless (no screen at all). Using the Web Inspector
locally is also possible when built with `ENABLE_WPE_PLATFORM` enabled.
```

WDYT?

Reply:
It's not a build time decision, but after thinking again maybe it's better not to mention the new api, since it's not officially released yet...

Reply:
Right, then let's land it as-is, and amend the documentation later on when the new API is deemed stable.